docs/src/manual/solutions.md

```diff
@@ -67,56 +67,50 @@ optimization solutions.
 
 ```jldoctest solutions; filter=r"[0-9]+\.[0-9]+e[\+\-][0-9]+"
 julia> solution_summary(model)
-* Solver : HiGHS
-
-* Status
-  Result count       : 1
-  Termination status : OPTIMAL
-  Message from the solver:
-  "kHighsModelStatusOptimal"
-
-* Candidate solution (result #1)
-  Primal status      : FEASIBLE_POINT
-  Dual status        : FEASIBLE_POINT
-  Objective value    : -2.05143e+02
-  Objective bound    : -2.05143e+02
-  Relative gap       : 1.38546e-16
-  Dual objective value : -2.05143e+02
-
-* Work counters
-  Solve time (sec)   : 6.01048e-04
-  Simplex iterations : 2
-  Barrier iterations : 0
-  Node count         : -1
+solution_summary(; result = 1, verbose = false)
+├ solver_name          : HiGHS
+├ Solution quality
+│ ├ termination_status : OPTIMAL
+│ ├ result_count       : 1
+│ ├ raw_status         : kHighsModelStatusOptimal
+│ └ objective_bound    : -2.05143e+02
+├ Solution (result = 1)
```

Comment:
If we call this `Solution` and not `Solution candidate` anymore then it feels like `Solution quality` is about this particular solution. It's not clear that `Solution` means a global thing in `Solution quality` while in `Solution (result = 1)` now it means one of the solutions

Reply:
I deleted "candidate" because it felt wrong. It wasn't a candidate. It was a solution. What about `s/Solution quality/Summary`? Or `Overall` or `Status`